README.md

```diff
@@ -46,7 +46,7 @@ cd gateway-proxy
 
 ## Usage
 
-The proxy can manage postage stamps for you in 3 modes of operation:
+The proxy can manage postage stamps for you in 4 modes of operation:
```

Comment:
So this feature is not about managing postage stamps, but is a completely separate feature, so this is actually not true. 

I would suggest to create another subsection that describe this feature separately from postage stamp management. Similarly to the "Bzz.link support"

Reply:
On the refactor I'll have all these issues resolved

Reply:
There are still 4 ways to deal with postage stamps with extending the TTL